CHANGELOG.md

```diff
@@ -124,6 +124,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - [Snapshot Interop] Add Changes in Create Snapshot Flow for remote store interoperability. ([#7118](https://github.com/opensearch-project/OpenSearch/pull/7118))
 - Add new query profile collector fields with concurrent search execution ([#7898](https://github.com/opensearch-project/OpenSearch/pull/7898))
 - Allow insecure string settings to warn-log usage and advise to migration of a newer secure variant ([#5496](https://github.com/opensearch-project/OpenSearch/pull/5496))
+- Add Lucene version to UploadedSegmentMetadata ([#8088](https://github.com/opensearch-project/OpenSearch/pull/8088))
```

Comment:
We may not need changelog entry. Refer: https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#changelog